source/jormungandr/tests/tests_mechanism.py

```diff
@@ -507,6 +507,7 @@ def deco(cls):
 
 class MockBssProvider(AbstractParkingPlacesProvider):
     def __init__(self, pois_supported, name='mock bss provider'):
+        AbstractParkingPlacesProvider.__init__(self)
```

Comment:
Don't know if this is mandatory, but this is cleaner.
However, this should be done on any class inheriting from AbstractParkingPlacesProvider :wink: 

Also same : use `super`